app/elements/submission/sequential/sequential-challenge.html

```diff
@@ -112,22 +121,20 @@
 					var currentTask = localStorage.getItem('currentTask');
 					var currentTaskIndex = localStorage.getItem('currentTaskIndex');
 					this.taskIndex = 0;
-					if(timer !== undefined && currentTask !== undefined && currentTaskIndex!==undefined){
+					if(timer !== null && currentTask !== null && currentTaskIndex !== null){
 						this.taskIndex = Number(currentTaskIndex);
 						this.loadTaskById(currentTask);
-						this.$.taskTimer.startWithOffset(timer);
+						this.$.taskTimer.startWithOffset(new Date(timer));
```

Comment:
AFAIK the timer will only use `date.getTime()`, so the two components could agree on passing a Unix timestamp and you could use the more lightweight `Date.now()` here.